src/primo/browzine-primo-adapter.js

```diff
@@ -659,6 +736,9 @@ browzine.primo = (function() {
     return template;
   };
 
+  //There's a bit of a question on whether we would need a retraction notice/eoc notice on this as we'd only get to this function
+  //if we don't already have a PDF link-we might only link to these live unpaywall responses if they have a direct-to-PDF link
+  //But we may want to investigate this to be sure...
```

Comment:
Unless you're thinking of investigating it, I think we can remove this comment... did you find an example of when this would get triggered?

You could ask Jay or Nick to see if they have an example of an article in Primo or Summon that would trigger a live Unpaywall call.  Then you could mark it as having an EOC and see how it behaves.  

If we want to investigate this later, let's create a new Jira describing what we're investigating and what question(s) we need to answer, and put the Jira number as a prefix to this comment like `TODO BZ-1234 - ` .  But if it seems like not a big deal, we can just remove this comment.

Reply:
@karlbecker Let me ping Jay and Nick and see. So essentially we need an example where we have a retracted or EOC article that doesn't have a pdf or article url, doesn't have an unpaywall pdf? I'm trying to think of how to formulate the question but admittedly I don't fully understand the flow of when we'd get to this function.

Reply:
@Evroble - I think that's right, yeah.  Quite a corner case. I wouldn't even ask about the details, I'd just ask if they know of any articles that require an Unpaywall call to show data.  If they can find one or two, then we can always mark it as EOC in the staging database for a little bit.

Reply:
What an investigation @Evroble and I did! We had to dive into the past, but here's a tl;dr: 

We identified that there's a bug that probably isn't triggered too often in the code... and we don't even have an article handy we can test it with.

Here's what we found:
1. If an API call is made, and that response has neither an article link nor a directToPdfUrl, but it does have a retracted flag on it, we will insert a retracted article link into search results thanks to this line of code that's been in there a long time: https://github.com/thirdiron/browzine-discovery-service-adapters/pull/127/files#diff-e538aa7cc112daa1deb5305ef7f4b9feda2970ad04ec12f05dc5b2dc2f9b05edR923
2. After that, the code will continue on to this line: `if ((request.readyState == XMLHttpRequest.DONE && request.status == 404) || (isArticle(scope) && (!directToPDFUrl && !articleLinkUrl && unpaywallUsable))) {`
3. Since that same article does not have direct to PDF or article link, it'll enter that if statement, and then if Unpaywall knows about it, we'll also make a new button there if unpaywall has a PDF
    * That's because the `unpaywallArticlePDFTemplate` and `unpaywallManuscriptPDFTemplate` functions get called, and they insert a link themselves at this point

So we should clean up this logic so only one or the other happens.

Great questions you asked Emily! 🙌